CIP-XXXX/README.MD

```diff
@@ -0,0 +1,164 @@
+
+Therefore, the following key principle will guide the balancing change process for wallets that are multi-delegating:
+
+Every multi-delegation transaction (creation or update) MUST include the delegation portfolio metadata specifying the percentage of funds they wish to delegate to each pool. This portfolio serves as a guideline for how funds SHOULD be distributed across different stake keys. 
+When subsequent transactions are built, input selection is performed as usual. The resulting change from the transaction is then evaluated for rebalancing. The structure and size of change outputs determined by the input selection algorithm SHALL be preserved.
+The change outputs SHALL be distributed as-is across multiple addresses associated with different stake keys, without altering the output values. Priority SHOULD be given to stake keys that are most divergent from the set preferences, thereby gradually realigning the actual distribution with the user's delegation strategy.
+
+#### Benefits and Implementation Considerations
+
+Consistent Delegation Strategy: Ensures that the user's delegation strategy is consistently maintained, even as funds are spent and rewards are received.
+
+UTXO Set Optimization: Maintains the health and efficiency of the UTXO set as intended by the input selection algorithm.
+
+Reduced Manual Adjustments: Minimizes the need for users to manually rebalance their stake across different pools, saving time and effort.
+
+### Reference Implementation
+
+An already working implementation can be found in Lace wallet and the Cardano-js-sdk:
+
+- Lace: https://github.com/input-output-hk/lace
+- Cardano JS SDK: https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/e2e/src/tools/multi-delegation-data-gen/index.ts
+
+## Rationale: how does this CIP achieve its goals?
+
+This CIP details a method for delegating to multiple pools from a single account, effectively meeting the Cardano community's demand for more versatile delegation strategies. By permitting users to manage multi-pool delegations within one account, this proposal significantly enhances the user experience, offering benefits in terms of investment diversification and risk management. Furthermore, it lays the groundwork for advanced wallet functionalities, like supporting multi-DRep delegation and introducing more complex algorithms for stake distribution. It is important to note, however, that this approach is designed within the constraints of the current Cardano features. Should future protocol developments introduce native multi-delegation capabilities, this CIP would be superseded.
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ] The proposal is adopted by all major wallets and dApps in the ecosystem.
```

Comment:
I think keeping this item as written would ensure that is stays in `Proposed` status forever.  Depending on the outcome of the discussion planned in my review above, this could be a sensible approach... although if other implementations are on the horizon then a more flexible acceptance threshold should be defined here.

Reply:
@AngelCastilloB - continued in https://github.com/cardano-foundation/CIPs/pull/628#issuecomment-2297894492 with a reason why it may actually make sense to leave this apparently unsatisfiable criterion on the list.